Trigger/src/ReadTriggerInfo_module.cc

```diff
@@ -1264,6 +1301,10 @@ namespace mu2e {
 	indexMother = 80;
       }else if (pdgM == 211){ //case 8: pi plus
 	indexMother = 90;
+      }else if (pdgM == 11){ //case 9: electrons
+	indexMother = 100;
+      }else if (pdgM == -11){ //case 10: positrons
```

Comment:
Please use an enum rather than than hard coded numbers.   Ask Dave if he supports an enum within BTrk and if it will be in the future KinKal.   If not, it is OK to use PDGCode::e_minus.  We moved PDGCode.hh out of MCDataProducts and into DataProducts so that it was no longer considered "MC" and could be used with data.  You should do this everywhere in the code base.

Reply:
I will also leave this to Dave to decide whether to do it now or to defer for a later PR

Reply:
Synchronizing pdgcodes across Offline, BTrk and KinKal is a bigger issue.  Fortunately they all use the same values.  It's clear though this code should use PDGCode values, not hardcoded numbers.

Reply:
I added the issue #292 and will mark it resolved.

Reply:
OK, no problem. Done.